src/BylawsText.js

```diff
@@ -0,0 +1,297 @@
+    </section>
+  )
+
+  const clubPurposeParagraph = (
+    <section id="clubPurpose">
+      <h3>§ 3 Zweck des Vereins</h3>
+      <p>
+        Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke
+        im Sinne des Abschnitts "Steuerbegünstigte Zwecke" der Abgabenordnung.
+        Zweck des Vereins ist {answers.clubPurpose}.
+      </p>
+    </section>
+  )
+
+  const nonProfitParagraph = (
+    <section id="nonProfit">
+      <h3>§ 4 Gemeinnützigkeit</h3>
+      <p>
+        Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie
+        eigenwirtschaftliche Zwecke.
+      </p>
+    </section>
+  )
+
+  const useOfFundsParagraph = (
+    <section id="useOfFunds">
+      <h3>§ 5 Mittelverwendung</h3>
+      <p>
+        Mittel des Vereins dürfen nur für die satzungsmäßigen Zwecke verwendet
+        werden. Die Mitglieder erhalten keine Zuwendungen aus Mitteln des
+        Vereins.
+      </p>
+    </section>
+  )
+
+  const membersParagraph = () => {
+    const members = join({
+      array: answers?.typeOfMembers?.filter(Boolean),
+      last: ' und ',
+    })
+
+    const typeOfRequest = join({
+      array: answers?.typeOfRequest?.filter(Boolean),
+      last: ' oder ',
+    })
+
+    return (
+      <section id="members">
+        <h3>§ 6 Erwerb der Mitgliedschaft</h3>
+        <p>
+          {members} können Vereinsmitglieder werden. Der Aufnahmeantrag kann{' '}
+          {typeOfRequest} gestellt werden. Über den Aufnahmeantrag entscheidet
+          der Vorstand.Gegen die Ablehnung, die keiner Begründung bedarf, steht
+          dem/der Bewerber/in die Berufung an die Mitgliederversammlung zu,
+          welche dann endgültig entscheidet.
+        </p>
+      </section>
+    )
+  }
+
+  const admissionFeeParagraph = () => {
+    return (
+      <section id="admissionFee">
+        <h3>§ 7 Aufnahmegebühr</h3>
+        {answers.admissionFee === 'yesAdmissionFee' ? (
+          <p>
+            Vor dem Eintritt wird eine Aufnahmegebühr fällig. Die Höhe bestimmt
+            die Mitgliederversammlung.
+          </p>
+        ) : (
+          <p>Eine Aufnahmegebühr wird nicht erhoben.</p>
+        )}
+      </section>
+    )
+  }
+
+  const memberFeeParagraph = () => {
+    return (
+      <section id="memberFee">
+        <h3>§ 8 Mitgliedsbeiträge</h3>
+        {answers.memberFee === 'yesMemberFee' ? (
+          <p>
+            Von den Mitgliedern werden Beiträge erhoben. Die Höhe der Beiträge
+            und deren Fälligkeit bestimmt die Mitgliederversammlung.
+          </p>
+        ) : (
+          <p>Mitgliedsbeiträge werden nicht erhoben.</p>
+        )}
+      </section>
+    )
+  }
+
+  const memberMeetingParagraph = () => {
+    const meetingDuties = join({
+      array: answers?.generalMeetingDuties?.filter(Boolean),
+      last: ' und ',
+    })
+
+    const getGeneralMeetingFrequency = (frequency) => {
+      if (frequency === 'monthly') return 'monatlich '
```

Comment:
I think an if else statement would be easier to read.

Reply:
as the answer possibilities being all "equal" and every if statement includes a return, I thought just an if-statement should be alright